### ISSUE TITLE
can not declare when existing configuration contains route domain decorations

### ISSUE THREAD

**Issue by jgruber** [F5Networks/f5-declarative-onboarding#19]
If I want to overwrite an existing configuration with a valid declaration and the existing configuration has route domains and route domain decorated L3 objects, a 500 error is returned as the existing configuration is schema checked as if DO was responsible for the existing configuration. That might not be the case. 

```
{"code":500,"status":"ERROR","message":"invalid config - rolled back","errors":["not a valid IPv4 address: 172.15.1.1%4091"],"declaration":{"async":true,"schemaVersion":"1.0.0","Common":{"hostname":"test14.sample.openstack.f5se.com","configsync":{"configsyncIp":"1.1.1.110","class":"ConfigSync"},"route_192_168_1_102_32":{"gw":"1.1.1.88","mtu":0,"class":"Route","network":"192.168.1.102/32"},"net_tape1439549_8c":{"interfaces":[{"tagged":false,"name":"1.1"}],"tag":4093,"class":"VLAN","mtu":1450},"selfip_tap1754743e_15":{"vlan":"net_tap1754743e_15","trafficGroup":"traffic-group-local-only","class":"SelfIp","allowService":"all","address":"172.15.1.103/24"},"selfip_tape1439549_8c":{"vlan":"net_tape1439549_8c","trafficGroup":"traffic-group-local-only","class":"SelfIp","allowService":"all","address":"1.1.1.110/24"},"dnsServers":{"nameServers":["8.8.8.8"],"search":["sample.openstack.f5se.com"],"class":"DNS"},"ntpServers":{"timezone":"UTC","class":"NTP","servers":["0.pool.ntp.org","1.pool.ntp.org"]},"selfip_tapf911cf4f_a8":{"vlan":"net_tapf911cf4f_a8","trafficGroup":"traffic-group-local-only","class":"SelfIp","allowService":"all","address":"172.14.1.109/24"},"route_192_168_233_0_24":{"gw":"1.1.1.102","mtu":0,"class":"Route","network":"192.168.233.0/24"},"net_tap1754743e_15":{"interfaces":[{"tagged":false,"name":"1.3"}],"tag":4091,"class":"VLAN","mtu":1500},"route_192_168_231_0_28":{"gw":"1.1.1.105","mtu":0,"class":"Route","network":"192.168.231.0/28"},"route_0_0_0_0_0_0_0_0":{"gw":"172.16.1.1","mtu":0,"class":"Route","network":"default"},"net_tapc5f0fbd1_fd":{"interfaces":[{"tagged":false,"name":"1.4"}],"tag":4090,"class":"VLAN","mtu":1500},"selfip_tapc5f0fbd1_fd":{"vlan":"net_tapc5f0fbd1_fd","trafficGroup":"traffic-group-local-only","class":"SelfIp","allowService":"all","address":"172.16.1.104/24"},"class":"Tenant","net_tapf911cf4f_a8":{"interfaces":[{"tagged":false,"name":"1.2"}],"tag":4092,"class":"VLAN","mtu":1500}},"class":"Device","label":"generated from OpenStack metadata"}}
```

**Comment by focrensh** [comment]
AT-114 for internal tracking

**Comment by dstokesf5** [comment]
It looks like this was resolved in the 1.6.0 release.
